package.json

```diff
@@ -101,7 +101,7 @@
     ]
   },
   "engines": {
-    "node": ">=16.x",
+    "node": ">=18.x",
```

Comment:
This should stay at >=16.x to keep NodeJS v16 support, rest of the changes are making sense, will proceed with this PR.

Reply:
Decided we're dropping node 16.x support